train_model.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
+import time
+from sklearn import linear_model
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.neighbors import KNeighborsClassifier
+import numpy as np
+from termcolor import colored
+import pickle
+import common
+
+start_time = time.time()
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--model_filename", default=model.pkl, dest="model_filename")
+parser.add_argument("--model", choices=["random_forest", "k_neighbors", "logistic_regression"], default=["random_forest"], dest="model")
+parser.add_argument("--train_dataset_size", default=100000, type=int, dest="train_dataset_size")
+args = parser.parse_args()
+print "Model used in this run is %s" % colored(args.model, "red")
+print "Train dataset size is %s" % colored(args.train_dataset_size, "red")
+
+train_X, train_y = load_dataset_from_file('train.csv', args.train_dataset_size, True)
+
+print "Input/Output time: %.3f" % (time.time() - start_time)
+train_start_time = time.time()
+
+if args.model == "logistic_regression":
+    model = linear_model.LogisticRegression(C=1, solver='lbfgs', 
```

Comment:
Nitpick: can you change your import from

`from sklearn import linear_model`

to

`from sklearn.linear_model import LogisticRegression` and use a simplified access here i.e.

`model = LogisticRegression(...)` instead of `model = linear.LogisticRegression)`.